src/main/react/app/routes/builder/context/node-context.tsx

```diff
@@ -26,6 +26,20 @@ export default function NodeContext({
     setCanSave(allValid)
   }
 
+  // Fills out input fields with already existing attributes when editing a node
+  useEffect(() => {
+    const currentAttributes = getAttributes(nodeId.toString())
+    if (currentAttributes && attributes) {
+      for (const [index, attribute] of attributes.entries()) {
+        const value = currentAttributes[attribute.name]
+        if (value && inputReferences.current[index]) {
+          inputReferences.current[index]!.value = value
```

Comment:
Is the non-null assertion operator really needed here? It should only be used in situations where it is not possible to fix otherwise, like in local variables.

```suggestion
        const currentInputReferance = inputReferences.current[index]
        if (value && currentInputReferance) {
          currentInputReferance.value = value
```

Reply:
Agreed! Oversight on my end

Reply:
No problem.